apps/deploy-web/src/context/CustomChainProvider/CustomChainProvider.tsx

```diff
@@ -13,6 +13,16 @@ import { akash, akashSandbox, akashTestnet, assetLists } from "@src/chains";
 import { useSelectedNetwork } from "@src/hooks/useSelectedNetwork";
 import { customRegistry } from "@src/utils/customRegistry";
 
+import "@interchain-ui/react/styles";
+import "@interchain-ui/react/globalStyles";
+
+let mm: any[] = [];
+if (typeof window !== 'undefined') {
+  import("@cosmos-kit/cosmos-extension-metamask").then((module) => {
+    mm = module.wallets;
+  });
+}
```

Comment:
I think a cleaner way would be to use next dynamic import, what do you think?
```
const metamask = dynamic(() => import("@cosmos-kit/cosmos-extension-metamask").wallets, {
  ssr: false
});
```
(I haven't tested this code)

Reply:
@baktun14 Tried that but it fails. Could not get it to work @baktun14